### PR TITLE
Split out reading of rest record types into separated sections

### PR DESCRIPTION
**Pull request by Honry**
***
<a href="https://pr-preview.s3.amazonaws.com/Honry/web-nfc/pull/297.html" title="Last updated on Aug 14, 2019, 3:00 PM UTC (e702a5d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/297/9ca0d40...Honry:e702a5d.html" title="Last updated on Aug 14, 2019, 3:00 PM UTC (e702a5d)">Diff</a>

### CODE REVIEW

**Comment by Honry** [comment]
@kenchris, @zolkis, PTAL, thanks!

**Comment by zolkis** [comment]
Looks good, but it would be nice if another pair of fresher eyes would also check it. :)

File: index.html [hunk 1]
```diff
@@ -3413,10 +3416,127 @@ <h3>Receiving and parsing content</h3>
...
   </p>
   </section>
 
+  <section><h3>Parsing NDEF MIME type records</h3>
+    <p>
+      To <dfn>parse a NDEF MIME type record</dfn> given a |ndefRecord| into a
+      |record:NDEFRecord|, run these steps:
+      <ol class=algorithm>
+        <li>
+          Let |MIME type| be the <a>MIME type</a>
```
Comment (kenchris) [inline index.html:3432]: Let's call it mimeType to not confuse with the <a>MIME type</a> and because of consistency

**Review by kenchris — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (kenchris) [reply index.html:3432]: We can do in a follow up

**Comment by Honry** [comment]
@kenchris, @zolkis, thanks for the review!

**Pull request merged**